build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python
```

Comment:
Is this recommended? I know that between versions of python there are often compatibility issues...